bundle/EventListener/HoneypotListener.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\SiteBundle\EventListener;
+
+use Netgen\InformationCollection\API\Events;
+use Netgen\InformationCollection\API\Exception\MissingAdditionalParameterException;
+use Netgen\InformationCollection\API\Value\Event\InformationCollected;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+class HoneypotListener implements EventSubscriberInterface
+{
+    public static function getSubscribedEvents(): array
+    {
+        return [
+            Events::INFORMATION_COLLECTED => ['onInformationCollected', 255],
```

Comment:
Since we depend on this (event - stop propagation) mechanism to execute the logic based on the honeypot field, please document it here.